assets/src/design-system/components/radio/radio.js

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useMemo } from 'react';
+import styled, { css } from 'styled-components';
+import { v4 as uuidv4 } from 'uuid';
+
+/**
+ * Internal dependencies
+ */
+import { themeHelpers, THEME_CONSTANTS } from '../../theme';
+import { FOCUS_VISIBLE_SELECTOR } from '../../theme/global';
+import { Text } from '../typography';
+
+const BORDER_WIDTH = 1;
+const TEXT_GAP_HEIGHT = 4;
+const RING_DIAMETER = 24;
+const BUTTON_CONTAINER_HEIGHT = 24 + BORDER_WIDTH * 2;
```

Comment:
^ `RING_DIAMETER` instead of 24?

Reply:
Looking at Figma then the total width/height seems to be `24` actually -- `22 + BORDER_WIDTH * 2`.

<img width="207" alt="Screenshot 2021-03-10 at 09 10 42" src="https://user-images.githubusercontent.com/3294597/110590489-2282fb00-8178-11eb-87fa-168bc22fbb1a.png">